github/repos.go

```diff
@@ -75,6 +75,7 @@ type Repository struct {
 	AssigneesURL     *string `json:"assignees_url,omitempty"`
 	BlobsURL         *string `json:"blobs_url,omitempty"`
 	BranchesURL      *string `json:"branches_url,omitempty"`
+	CodeOfConduct    *string `json:"code_of_conduct,omitempty"`
```

Comment:
Pretty sure you want the type to be `*CodeOfConduct`, not `*string`. ;)

This is the sample value in GitHub API docs:

![image](https://cloud.githubusercontent.com/assets/1924134/26750930/cfb90c48-47fb-11e7-8601-7105a3976298.png)